src/electron/main.ts

```diff
@@ -27,15 +27,14 @@ export default class Main {
     const __dirname = path.dirname(__filename);
 
     Main.mainWindow = new Main.BrowserWindow({
-      width: 800,
+      width: 1200,
```

Comment:
Nice! Had changed it as well on local branch after the change to auto open the dev tools